src/StylePlugin.js

```
@@ -427,7 +480,7 @@ function getAnExistingAttribute(schema) {
 
   try {
     existingAttr = schema['marks']['link']['attrs']['href'];
-  } catch (err) {}
+  } catch (err) { }
```

Provide a reason for this try block with an empty catch handler.

---

This is an unnessasary code, shall correct this.